src/Domain/CoordinateMaps/EquiangularMap.hpp

```diff
@@ -0,0 +1,97 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class EquiangularMap.
+
+#pragma once
+
+#include <array>
+#include <memory>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMapsGroup
+ * \brief Non-linear map from \f$\xi \in [A, B]\rightarrow x \in [a, b]\f$.
+ *
+ * The formula for the mapping is...
+ * \f[
+ * x = \frac{a}{2} (1-\mathrm{tan}(\frac{\pi(2\xi-B-A)}{4(B-A)}) +
+ *     \frac{b}{2} (1+\mathrm{tan}(\frac{\pi(2\xi-B-A)}{4(B-A)})\\
+ * \xi =\frac{A}{2} (1-\frac{4}{\pi}\mathrm{arctan}(\frac{2x-a-b}{b-a}))+
+ *      \frac{B}{2} (1+\frac{4}{\pi}\mathrm{arctan}(\frac{2x-a-b}{b-a}))
+ * \f]
+ *
+ *\note The intermediate step in which a tangent map is applied can be more
+ * clearly understood if we define the coordinates:
+ * \f[\xi_{logical} := \frac{2\xi-B-A}{B-A} \in [-1, 1]\\
```

Comment:
Use consistent formatting of equations. That is, `\f[` newline, `equation stuff` newline `\f]`. Again, check rendered, I think `align` with `&` alignment will be better